src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/ViewModels/GameReports/StandingsEntryViewModel.cs

```diff
@@ -158,8 +156,32 @@ public static List<StandingsEntryViewModel> SetPositions(List<StandingsEntryView
         public static bool EntriesHaveSamePosition(StandingsEntryViewModel firstEntry, StandingsEntryViewModel secondEntry)
         {
             return firstEntry.Points == secondEntry.Points
-                && firstEntry.SetsRatio == secondEntry.SetsRatio
-                && firstEntry.BallsRatio == secondEntry.BallsRatio;
+                && EqualsDigitPrecision(firstEntry.SetsRatio, secondEntry.SetsRatio, TOLERANCE)
+                && EqualsDigitPrecision(firstEntry.BallsRatio, secondEntry.BallsRatio, TOLERANCE);
+            //return true;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)